packages/@react-native-windows/cli/src/index.ts

```diff
@@ -59,23 +88,64 @@ export async function generateWindows(
   ns: string,
   options: GenerateOptions,
 ) {
-  if (!fs.existsSync(projectDir)) {
-    fs.mkdirSync(projectDir);
-  }
+  let error;
+  try {
+    if (!fs.existsSync(projectDir)) {
+      fs.mkdirSync(projectDir);
+    }
+
+    installDependencies(options);
+
+    const rnwPackage = path.dirname(
+      require.resolve('react-native-windows/package.json', {
+        paths: [projectDir],
+      }),
+    );
+    const templateRoot = path.join(rnwPackage, 'template');
+    await copyProjectTemplateAndReplace(
+      templateRoot,
+      projectDir,
+      name,
+      ns,
+      options,
+    );
+  } catch (e) {
+    error = e;
+    telClient.trackException({ exception: error });
```

Comment:
Need to be careful to see what all this tracks. We've seen issues before where diagnostic information can unintentionally leak PII when paths are involved. E.g. if the exception has a stack trace with file paths on the users machine.

Reply:
Thanks for bringing this up.  I'm also really concerned about PII leakage here - even the name of my project can be something I consider a PII leak if I'm working on something confidential and unannounced, so I'd also double-check if there are other places where project name/path may get tracked inadvertently.

Reply:
I'll scrub the stacks to preserve only method name, line number, file name and leaf directory. These will never contain any PII.

![image](https://user-images.githubusercontent.com/22989529/97774636-d1eaf980-1b16-11eb-8189-a91c2cc53817.png)